CHANGELOG.md

```diff
@@ -13,7 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add configurable pause annotation support.
 
+## Updated
 
+- Updated Kubernetes dependencies to v1.18.5.
```

Comment:
```suggestion
- Updated backward incompatible Kubernetes dependencies to v1.18.5.
```

Reply:
Thanks Pawel that's a good shout. A lot of the libraries already got merged but there will be a docs PR for updating operators etc. I'll use this there.